06-find.md

```diff
@@ -102,25 +102,51 @@ $ grep -n it haiku.txt
 
 Here, we can see that lines 5, 9, and 10 contain the letters "it".
 
-We can combine flags as we do with other Unix commands.
-For example,
-since `-i` makes matching case-insensitive and `-v` inverts the match,
-using them both only prints lines that *don't* match the pattern
-in any mix of upper and lower case:
+We can combine options (i.e. flags) as we do with other Unix commands.
+For example, let's find the lines that contain the word 'the'.
```

Comment:
"that contain the word 'the'." -> "that contain the word "the"."

The lesson is using double quote so we should use it here. =)